CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ Compatibility:
 * Set `RbConfig::CONFIG['configure_args']` for openssl and libyaml (#3170, #3303, @eregon).
 * Support `Socket.sockaddr_in(port, Socket::INADDR_ANY)` (#3361, @mtortonesi).
 * Implement the `Data` class from Ruby 3.2 (#3039, @moste00, @eregon).
+* Implement `MatchData` methods `byteoffset`, `deconstruct`, and `deconstruct_keys` from Ruby 3.2 (@rwstauner).
```

Comment:
```suggestion
* Implement `MatchData` methods `byteoffset`, `deconstruct`, and `deconstruct_keys` from Ruby 3.2 (#3039, @rwstauner).
```

Reply:
I'd also change to `MatchData#{byteoffset,deconstruct,deconstruct_key}` to be consistent with the rest of the changelog.